earthfile2llb/listener.go

```diff
@@ -662,6 +667,12 @@ func (ssf *StringSliceFlag) Set(arg string) error {
 	return nil
 }
 
+var shellNameStringRegexp = regexp.MustCompile("^[a-zA-Z_]+[a-zA-Z0-9_]*")
+
+func checkShellName(str string) string {
```

Comment:
Mind renaming this to `checkEnvVarName`? "Shell name" could be misunderstood as the shell itself.